### ISSUE TITLE
batect gets stuck when downloading old versions

### ISSUE THREAD

**Issue by nkrul** [batect/batect#969]
### Describe the bug

An old project with a fresh checkout from git.
>batect
Downloading Batect version 0.68.0 from https://dl.bintray.com/batect/batect/0.68.0/bin/batect-0.68.0.jar...
Downloading failed with error: The remote server returned an error: (403) Forbidden.

### Steps to reproduce

run batect 0.68.0.

### Expected behaviour

batect runs
Specifically, batect is able to download the current jar.

Alternatively, batect could suggest running --upgrade on a 403 response (and not require the jar when running --upgrade)

### Actual behaviour

Get a 403 error from bintray

### Version information

```plain text
>batect --version
Downloading Batect version 0.68.0 from https://dl.bintray.com/batect/batect/0.68.0/bin/batect-0.68.0.jar...
Downloading failed with error: The remote server returned an error: (403) Forbidden.
```

### Any other information

perhaps a fallback to github? 
or embedding the upgrade logic in the launcher script

**Comment by charleskorn** [comment]
Hi @nkrul, thanks for the report. Unfortunately there's not much I can do here. I can't go back and change the wrapper script in old versions as users have already downloaded a copy and so they wouldn't get the changes. 

[This blog post from March](https://batect.dev/blog/2021/03/10/bintray-shutdown) explains some of the options available to deal with the shutdown of Bintray. Newer versions of Batect use a service that I control (updates.batect.dev), so a change like this in the future shouldn't have an impact on users or projects using an older version.

**Comment by nkrul** [comment]
No problem with not updating old versions. I did chuckle that I had v0.68 and its fixed in v0.69.
Thanks for the response
